servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/ContextPreservingSingleSubscriber.java

```diff
@@ -18,22 +18,24 @@
 import io.servicetalk.concurrent.Cancellable;
 import io.servicetalk.concurrent.SingleSource;
 import io.servicetalk.concurrent.SingleSource.Subscriber;
-import io.servicetalk.context.api.ContextMap;
 
 import javax.annotation.Nullable;
 
 import static java.util.Objects.requireNonNull;
 
 class ContextPreservingSingleSubscriber<T> implements Subscriber<T> {
```

Comment:
Consider making this class `final`, removing `final` modifier from method, and removing `invokeOnSubscribe` method to make it consistent with `ContextPreservingCompletableSubscriber`